libpdbg/dtb.c

```diff
@@ -277,8 +330,20 @@ static void bmc_target(struct pdbg_dtb *dtb)
 	case CHIP_ID_P10:
 		pdbg_log(PDBG_INFO, "Found a POWER10 OpenBMC based system\n");
 		pdbg_proc = PDBG_PROC_P10;
-		if (!dtb->backend.fdt)
-			dtb->backend.fdt = &_binary_bmc_kernel_dtb_o_start;
+		if (!dtb->backend.fdt) {
+			char* system_type = get_p10_system_type();
+			if (strcmp(system_type, "Everest") == 0) {
+				pdbg_log(PDBG_INFO, "bmc_target - loading bmc kernel everest target");
+				dtb->backend.fdt = &_binary_bmc_kernel_everest_dtb_o_start;
+			} else if (strcmp(system_type, "Rainier 2U") == 0 || strcmp(system_type, "Rainier 4U") ==0 ) {
```

Comment:
A space would be nice `strcmp(system_type, "Rainier 4U") ==0` to `strcmp(system_type, "Rainier 4U") == 0`